packages/view-types/heatmap/src/HeatmapOptions.js

```diff
@@ -47,6 +53,20 @@ export default function HeatmapOptions(props) {
           </OptionSelect>
         </TableCell>
       </TableRow>
+      <TableRow>
+        <TableCell className={classes.labelCell}>
+          Disable Tooltip
+        </TableCell>
+        <TableCell className={classes.inputCell}>
+          <Checkbox
+            className={classes.checkbox}
+            value={tooltipDisabled}
```

Comment:
```suggestion
            checked={tooltipDisabled}
```
The checkbox state is not persisting with `value` - For reference,  https://v4.mui.com/api/checkbox/ 

Ditto for Spatial/Embedding options component

Reply:
I didn't notice the problem with persistence, thanks! I have updated all checkbox components and also added a comment for the future.